.github/workflows/build-release.yml

```diff
@@ -35,14 +35,27 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install wheel setuptools
+        pip install wheel setuptools rjsmin rcssmin
     
     - name: Build tar.gz
       run: python3 setup.py sdist
     
     - name: Build wheel
-      run: python3 setup.py bdist_wheel
-    
+      run: |
+        bundlejs="$(sha224sum assets/scripts/*.js | sha224sum | cut -d ' ' -f1).js"
+        bundlecss="$(sha224sum assets/styles/*.css | sha224sum | cut -d ' ' -f1).css"
+        sed '/<script.*src=/d' bin/views/*.html
+        sed "/<link.*rel=[\"']?stylesheet/d" bin/views/*.html
+        echo "<script type=javascript src=\"/assets/${bundlejs}\" defer></script>" >> bin/views/head.html
+        echo "<link rel=stylesheet href=\"/assets/${bundlecss}\">" >> bin/views/head.html
```

Comment:
Human signification : append both the JS and CSS bundle to the `head.html` file (the file that is enclosed in all files)